pkg/xray/resource_xray_repository_config.go

```diff
@@ -47,29 +60,75 @@ func resourceXrayRepositoryConfig() *schema.Resource {
-			Description:   `Single repository configuration. Only one of 'config' or 'paths_config' can be set.`,
+			Description:   "Single repository configuration. Only one of 'config' or 'paths_config' can be set.",
 			ConflictsWith: []string{"paths_config"},
 			Elem: &schema.Resource{
 				Schema: map[string]*schema.Schema{
 					"vuln_contextual_analysis": {
 						Type:        schema.TypeBool,
 						Optional:    true,
-						Description: `Only for SaaS instances, will be available after Xray 3.59. Enables vulnerability contextual analysis.`,
+						Description: "Only for SaaS instances, will be available after Xray 3.59. Enables vulnerability contextual analysis.",
 					},
 					"retention_in_days": {
 						Type:             schema.TypeInt,
 						Optional:         true,
 						Default:          90,
-						Description:      `The artifact will be retained for the number of days you set here, after the artifact is scanned. This will apply to all artifacts in the repository.`,
+						Description:      "The artifact will be retained for the number of days you set here, after the artifact is scanned. This will apply to all artifacts in the repository.",
 						ValidateDiagFunc: validator.IntAtLeast(0),
 					},
+					"exposures": {
+						Type:        schema.TypeSet,
+						Optional:    true,
+						MinItems:    1,
+						MaxItems:    1,
+						Description: "Enables Xray to perform scans for multiple categories that cover security issues in your configurations and the usage of open source libraries in your code. Available only to CLOUD (SaaS)/SELF HOSTED for ENTERPRISE X and ENTERPRISE+ with Advanced DevSecOps",
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"scanners_category": {
+									Type:     schema.TypeSet,
+									Optional: true,
+									MinItems: 1,
+									MaxItems: 1,
+									Elem: &schema.Resource{
+										Schema: map[string]*schema.Schema{
+											"malicious_code": {
+												Type:        schema.TypeBool,
+												Optional:    true,
+												Description: "",
+											},
+											"services": {
+												Type:        schema.TypeBool,
+												Optional:    true,
+												Description: "Detect whether common OSS libraries and services are configured securely, so application can be easily hardened by default.",
+											},
+											"secrets": {
+												Type:        schema.TypeBool,
+												Optional:    true,
+												Description: "Detect any secret left exposed in any containers stored in Artifactory to stop any accidental leak of internal tokens or credentials.",
+											},
+											"iac": {
+												Type:        schema.TypeBool,
+												Optional:    true,
+												Description: "Scans IaC files stored in Artifactory for early detection of cloud and infrastructure misconfigurations to prevent attacks and data leak. Only support by Terraform Backend package type.",
+											},
+											"applications": {
```

Comment:
Should we have defaults for all booleans here? And remove `omitempty` in the struct?